node-gui/src/main_window/main_widget/tabs/mod.rs

```diff
@@ -13,104 +13,132 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use iced::{
-    alignment,
-    widget::{Column, Container},
-    Command, Element, Font, Length,
-};
+use iced::{Command, Element};
 use iced_aw::{TabLabel, Tabs};
 
-use crate::backend_controller::NodeBackendController;
+use crate::{
+    backend::{messages::WalletId, BackendSender},
+    main_window::NodeState,
+};
 
 use self::{
+    network::{NetworkMessage, NetworkTab},
     settings::{SettingsMessage, SettingsTab, TabBarPosition},
     summary::{SummaryMessage, SummaryTab},
+    wallet::{WalletMessage, WalletTab},
 };
 
+pub mod network;
 pub mod settings;
 pub mod summary;
-
-const TAB_PADDING: u16 = 16;
-
-const ICON_FONT: Font = iced::Font::External {
-    name: "Icons",
-    bytes: include_bytes!("../../../../fonts/icons.ttf"),
-};
-
-enum Icon {
-    User,
-    CogAlt,
-}
-
-impl From<Icon> for char {
-    fn from(icon: Icon) -> Self {
-        match icon {
-            Icon::User => '\u{E800}',
-            Icon::CogAlt => '\u{E802}',
-        }
-    }
-}
+pub mod wallet;
 
 #[derive(Debug, Clone)]
 pub enum TabsMessage {
-    Start,
     TabSelected(usize),
+    WalletAdded(WalletId),
+    WalletRemoved(WalletId),
     Summary(SummaryMessage),
+    Network(NetworkMessage),
     Settings(SettingsMessage),
+    WalletMessage(WalletId, WalletMessage),
 }
 
 pub struct TabsWidget {
     active_tab: usize,
     summary_tab: SummaryTab,
+    network_tab: NetworkTab,
```

Comment:
networking

We use the word "networking" because "network" can be confused with the network being mainnet, testnet, etc.